src/utils/parsers/common.ts

```diff
@@ -330,15 +350,16 @@ export abstract class BaseParser {
 
     parseFile(file: usage.File): VariableUsageMatch[] {
         const result: VariableUsageMatch[] = []
+        const filteredFile = this.getFilteredFile(file)
         let fileContent = ''
-        for (const line of file.lines) {
-            fileContent = fileContent.concat(line.content.trim()) 
+        for (const line of filteredFile.lines) {
+            fileContent = fileContent.concat(line.content)
```

Comment:
oh! I think dropping the `trim` is causing the extra lines (since `extractUsageInformation` is still trimmin)

Reply:
i'm a dummy